### PR TITLE
Made changes to De, Dem, Dom, Det, Den and description look-up script

### PR DESCRIPTION
**Pull request by viremn**
I have implemented Lars' script for adjusting lemmas and features for words [den, det, de, dem, dom]. As it works right now, a specific field of a token line is updated to a specific value stored in the config dictionary found in changes_by_upos_config.py, if both the token's form and upos correspond to the keys in the dictionary. It is possible to add any number of token forms/upos combinations to the dictionary together with their respective changes. If we need to make similar changes for other words in the future (only dependent on upos and form), these changes can be added in the config dictionary. This script can be slotted in the mamba to conllu pipeline before the train-dev-test split. 

I have also implemented Lars' script for extracting all descriptions (lemma, upos, feats, etc.) for a list of token forms (get_descriptions.py). It can either be run with one or more words as system arguments, or with a .txt file as input.

### CODE REVIEW

**Comment by jnivre** [comment]
This all sounds good, but I am wondering about the .conllu files included in the pull request. Normally, .conllu files should not appear in the not-to-release subfolder but only in the top folder. Also, are these the conllu files resulting after applying the new scripts? And did you verify first that you can replicate the old files?

I think it might be best to separate this into two pull requests, one adding the new scripts to the not-to-release subfolder and one adding the .conllu files to the top folder (so that I can review the changes to the annotations before merging the request).

File: not-to-release/__pycache__/changes_by_upos_config.cpython-39.pyc
Comment (dan-zeman) [inline not-to-release/__pycache__/changes_by_upos_config.cpython-39.pyc:1]: Why should pycache be versioned on Github?

Comment (jnivre) [reply not-to-release/__pycache__/changes_by_upos_config.cpython-39.pyc:1]: Thanks, Dan. This is work in progress. Victor is a new RA in Uppsala helping me to fix things in the Swedish treebanks. Bear with us.

Comment (viremn) [reply not-to-release/__pycache__/changes_by_upos_config.cpython-39.pyc:1]: Hi Dan. Thank you for the comment. Indeed, there is not reason for it to be there. I will see that that it is not included in the future!

**Comment by viremn** [comment]
> This all sounds good, but I am wondering about the .conllu files included in the pull request. Normally, .conllu files should not appear in the not-to-release subfolder but only in the top folder. Also, are these the conllu files resulting after applying the new scripts? And did you verify first that you can replicate the old files?
> 
> I think it might be best to separate this into two pull requests, one adding the new scripts to the not-to-release subfolder and one adding the .conllu files to the top folder (so that I can review the changes to the annotations before merging the request).

These conllu files are the result of the script, yes, and I have also verified that the script reproduces the old files (when the new addition to the pipe line is excluded). 

Should I remove this pull request then and make two new ones, one with the new scripts and one with the generated files in the directory above?

**Comment by viremn** [comment]
I removed the output and the pycache from the pull request. Should I do another commit to this pull request with the updated conllu files in the top folder (replacing the old ones) or should I do a separate pull request for them? @jnivre

**Comment by jnivre** [comment]
I think it’s best to do two separate requests.

**Pull request merged**